customers-service/lombok.config

```diff
@@ -0,0 +1 @@
+lombok.addLombokGeneratedAnnotation = true
```

Comment:
Not urgent: Where is this file located? Would this not be something that would automatically be added when the dependency is added to project?

Reply:
No, it was added manually. This allows our jacoco to recognize generated code from Lombok annotations and then not require then in coverage.